corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_ko_templates.html

```diff
@@ -228,6 +228,12 @@ <h4 class="panel-title panel-title-nolink">
         <th class="col-sm-1"></th>
         <th data-bind="class: saveOnlyEditedFormFieldsEnabled ? 'col-sm-3' : 'col-sm-4'">
           {% trans "Case Property" %}
+          <span class="hq-help-template"
```

Comment:
Do you know about the `makeHqHelp` knockout binding? I'd think that would work here.

Reply:
Thanks. I've updated the PR to use that, which should simplify this a lot